apps/core/components/passengerForm/DateInput.js

```diff
@@ -0,0 +1,131 @@
+// @flow
+
+import React from 'react';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import { format, setMonth } from 'date-fns';
+import { View } from 'react-native';
+import {
+  StyleSheet,
+  Picker,
+  Text,
+  TextInput,
+} from '@kiwicom/universal-components';
+
+type Props = {|
+  +onBirthDateSubmit: Date => void,
```

Comment:
What about using something more universal like `onDateChange`. I think that there is some possibility that we will use this component for some different purpose than just birth date.

Reply:
True. Thanks!